package/gluon-core/files/usr/bin/gluon-reload-daemons

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+cd "/lib/gluon/reload.d" || exit 1
+
+err=0
+
+for service in *; do
+	echo "restarting: ${service} ..."
+	exec "/etc/init.d/$service restart" || err=1
```

Comment:
Uhm... did you test this?

"exec" will replace the calling shell, so if this command could succeed, only the first restart would happen. It cannot succeed though, as the shell will consider `/etc/init.d/$service restart` the name of the binary to run instead of a shell command with an argument due to the quotation marks.

Generally, I find this approach questionable. Restarting daemons is already an "expensive" operation, so it wouldn't hurt to make this more flexible by making `/lib/gluon/reload.d/*` shell scripts and just run the scripts (similar to our upgrade scripts).

Also, we don't always want to `restart` a daemon, a `reload` will sometimes suffice.